commit 7c70fee5ae5c4fb542e432599cb85c8031b952a0
Author: Sara Sharon <sara.sharon@intel.com>
Date:   Tue Feb 2 11:55:53 2016 +0200

    iwlwifi: mvm: extend time event duration
    
    Before authentication, we start a time event during
    which we wait for a beacon in order to sync our timers.
    If we didn't hear the beacon during this time - we abandon
    the connection. However, in congested environment, it was
    observed we might not hear beacons in that time slot.
    Extend the time event to give the connection a better chance.
    
    Signed-off-by: Sara Sharon <sara.sharon@intel.com>
    Signed-off-by: Emmanuel Grumbach <emmanuel.grumbach@intel.com>

diff --git a/drivers/net/wireless/intel/iwlwifi/mvm/time-event.h b/drivers/net/wireless/intel/iwlwifi/mvm/time-event.h
index 99d9a35ad5b1..3d2e8b6159bb 100644
--- a/drivers/net/wireless/intel/iwlwifi/mvm/time-event.h
+++ b/drivers/net/wireless/intel/iwlwifi/mvm/time-event.h
@@ -115,7 +115,7 @@
  * needed by the driver.
  */
 
-#define IWL_MVM_TE_SESSION_PROTECTION_MAX_TIME_MS 500
+#define IWL_MVM_TE_SESSION_PROTECTION_MAX_TIME_MS 600
 #define IWL_MVM_TE_SESSION_PROTECTION_MIN_TIME_MS 400
 
 /**